components/slices/Text.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react'
 import { RichText } from 'prismic-reactjs'
-import { linkResolver } from 'prismic-configuration'
-import htmlSerializer from 'utils/htmlSerializer'
+import { linkResolver, customLink } from 'prismic-configuration'
 
 const Text = ({ slice }) => (
   <div className='post-part single container'>
-    {RichText.render(slice.primary.text, linkResolver, htmlSerializer)}
+    <RichText render={slice.primary.text} linkResolver={linkResolver} serializeHyperlink={customLink} />
```

Comment:
Links to the web and links to a media aren't working. They take you to the homepage.